.gitignore

```diff
@@ -9,6 +9,7 @@ writeable_dir/
 # Files written by jobs
 *techsupport_ooo.json
 logs
+workspace/workflows/cache.json
```

Comment:
`*cache.json` should gitignore it anywhere, irrespective of where `settings.WRITEABLE_DIR` is (see other comment)

Reply:
Perhaps `workflows_cache.json` would be a better name, given that we can't keep it alongside the workflows code.

Reply:
I think maybe this line can just be removed. It seems that `writeable_dir/`is ignored anyway, which should cover production and the default local dev setting. 
A local dev that has altered  `WRITEABLE_DIR` in their `.env` will be able to commit it, but I assume they would know what they are doing(?)

Reply:
In doing this I realised the tests for main() had been writing empty cache files, so this is fixed

Reply:
writeable_dir/ is ignored, but you can change your local env settings, so it could be somewhere else. I'd incude it just in case